events/message.js

```diff
@@ -20,8 +20,10 @@ module.exports = (client, message) => {
 		if (message.content.indexOf(client.config.prefix) !== 0) return;
 	}
 
+	let trustrole = message.guild.roles.find(role => role.name === "Developer");
```

Comment:
You need a check for servers where 'Developer' role does not exist, and probably some sort of check for 'DEVELOPER' and 'developer'.  It may be better to attach it to the 'manage channel' permission?